src/CalcManager/CalculatorHistory.cpp

```diff
@@ -7,8 +7,7 @@
 using namespace std;
 using namespace CalculationManager;
 
-CalculatorHistory::CalculatorHistory(CALCULATOR_MODE eMode, size_t maxSize) :
-    m_mode(eMode),
+CalculatorHistory::CalculatorHistory(size_t maxSize) :
```

Comment:
@danbelcher-MSFT - Can you check the archived history to see if the fact that this is still here is the bug, or the fact that the code referencing it is absent is the bug?

Reply:
The member variable exists for this class as far back as we have archived history.  It was never used from the earliest record.  Seems safe enough to remove.  I believe CalculatorManager does the logic of checking if the current mode supports history or not.